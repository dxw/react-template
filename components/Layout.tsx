import React, { ReactNode } from "react";

type LayoutProps = {
  children: ReactNode;
};

const Layout = ({ children }: LayoutProps): JSX.Element => (
  <div>{children}</div>
);

export default Layout;
