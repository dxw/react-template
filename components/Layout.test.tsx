/* eslint-env jest */
import React from "react";
import renderer from "react-test-renderer";

import Layout from "./Layout";

it("renders correctly", () => {
  const component = renderer.create(
    <Layout>
      <p>Test content</p>
    </Layout>
  );

  expect(component.toJSON()).toMatchSnapshot();
});
