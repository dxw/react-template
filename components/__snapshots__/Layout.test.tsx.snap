// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p>
    Test content
  </p>
</div>
`;
